src/alfasim_sdk/_internal/alfacase/generate_case_description_docstring.py

```diff
@@ -192,7 +195,7 @@ def _get_class_with_reference(visible_name: str, ref: str) -> str:
     """
     Return the name of the class with a valid reference to be used by sphinx.
     """
-    return f"\\ :class:`{visible_name} <{ref}>`\\"
+    return f"\\ :class:`{visible_name} <{ref}>`"
```

Comment:
I believe that this scape character at the end is necessary, otherwise, the docs are not formatted correctly (see the screenshot).
```suggestion
    return f"\\ :class:`{visible_name} <{ref}>`\\"
```
![image](https://user-images.githubusercontent.com/5083518/114933097-b2b6fe80-9e0e-11eb-8905-143dbff2b9b9.png)

Reply:
We should have a test checking the rendered stuff.
Or is that impractical?

Reply:
IMHO, we already have a render test, just look at the number of files that were changed.
The issue here was a modification that affected several files and we didn't check the output, just ran `--force-regen`.

A visual render test is possible but in my opinion, would lead to the same outcome (executing `--force-regen` on a massive change), but now with more dependencies like starting a browser and so on.

What are your thoughts on this @oliveria, @bruno.

Reply:
Yes, we will force regen...
But the faulty result will be visible in the PR and not force someone to check the full generated documentation and manually compare it with the "master branch" one.
I am not saying we need to test all rendered files, just some, one for each "hack" we need top get the proper final rendering.